run.sh

```diff
@@ -19,12 +20,17 @@ function start_container() {
     USER_OPTS="--env=PLEX_UID=${PLEX_UID} --env=PLEX_GID=${PLEX_GID}"
   fi
 
+  # what version should we pass to the container?
+  if [ ${PLEX_SERVER_VERSION} == "latest" ]; then
```

Comment:
i like this idea, but there are two types of latest: 1) public and (2) plex pass. we should probably make this `public` or something similar so we can leave ourselves open to implement plex pass latest. thoughts?